### ISSUE TITLE
ModSecurity2 write LogsParts not allowed

### ISSUE THREAD

**Issue by devnull-hub-lab** [SpiderLabs/ModSecurity#2744]
Hi,

I removed section E and F from SecAuditLogParts:
SecAuditLogParts ABIJDHZ

But after a block in my wordpress page, modsecurity.log generate E Section (--a49cd13e-E--), according E6u.txt attached:
[E6u.txt](https://github.com/SpiderLabs/ModSecurity/files/8732641/E6u.txt)

But I made simple test, like /phpfilethatdonotexist.php?something=../../etc/passwd, and the Section "E" isn't written.

**Server:**
 - ModSecurity 2.9.5
 - WebServer: Apache HTTPD Server 2.4.53
 - OS: FreeBSD 13.1

**Rule Set:**
 - Last from GIT: https://github.com/coreruleset/coreruleset.git

**Comment by martinhsv** [comment]
Hello @devnull-hub-lab ,

Even though you have that setting for SecAuditLogParts, what you are seeing could be because of a "ctl:auditLogParts=+E" action for one of the rules triggered for that HTTP transaction.

In your case rule 953110 is one of the rules listed in the audit log that you attached. And in the version of that rule that I just checked at their github site, it does indeed include "ctl:auditLogParts=+E"

**Comment by devnull-hub-lab** [comment]
Hi @martinhsv,

Thanks for your reply, and sorry for any inconvenience.

I understand this, but I not agree with this "ctl:auditLogParts=+E" in the rules.
They are bypassing what was defined in modsecurity.conf, which should be mandatory for logging all rules.

Maybe if there was a directive in crs-setup.conf where it could be defined which LogParts is mandatory...
I removed "ctl:auditLogParts=+E" but don't worked. But there are several other places with this directive, it's probably going through another point. I'll do a fine analysis on the rules.

Thanks again, Martin!

***Status update: I removed all ctl:auditLogParts=+E for testing on all .conf rules, and worked!
